src/services/datasources/services/datasources.js

```diff
@@ -5,7 +5,7 @@ const {
 	iff, isProvider, validateSchema, disallow,
 } = require('feathers-hooks-common');
 const { datasourceModel } = require('../model');
-const { updatedBy, createdBy, protectFields } = require('../hooks');
+const { updatedBy, createdBy, protectFields, validateParams } = require('../hooks');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected a line break before this closing brace.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4993300)